include/compressed_storage.h

```diff
@@ -23,13 +24,18 @@ class CompressedDataStorage
     const int num_cols();
 
     const std::vector<int>& row_ptr();
-    const std::vector<int>& col_idx();
+    const std::vector<int>& col_idx() const;
```

Comment:
Any reason to have this here, but not on `row_ptr` and `data`?

Reply:
I would prefer to mark all 3 of them const, because it would require less changes in the code (just removing const will require some more lines to change)